docs/source/install/profile.rst

```diff
@@ -18,6 +18,23 @@ Ideally, store that file directly in your ``$HOME/`` and source it after connect
 
 We listed some example ``picongpu.profile`` files below which can be used to set up PIConGPU's dependencies on various HPC systems.
 
+Hemera (HZDR)
+-------------
+
+For this profile to work, you need to download the :ref:`PIConGPU source code <install-dependencies-picongpu>` manually.
+
+Queue: defq (2x Intel Xeon Gold 6148, 20 Cores + 20 HyperThreads/CPU)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
these need to be aligned in `.rst` with the length of the headline

Reply:
ahhh shit I forget each time that it must have the correct lenght

Reply:
Haha, yes. I also did not see it immediately, sorry!